### ISSUE TITLE
Basic auth is not supposed to accept colon ":" in username

### ISSUE THREAD

**Issue by hsablonniere** [ahmadnassri/app-restconsole#69]
I'm not sure of that but it seems that colon character can't be part of a username...

http://www.ietf.org/rfc/rfc2617.txt

**Comment by ahmadnassri** [comment]
Good catch, I didn't do much validation in version, will add to the next.
